tests/StatsdClient.Tests/GlobalSetup.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using NUnit.Framework;
+using StatsdClient;
+
+[SetUpFixture]
+public class GlobalSetup
+{
+    private static readonly string[] EnvironmentVariables =
+    {
+        StatsdConfig.ServiceEnvVar, StatsdConfig.EnvironmentEnvVar, StatsdConfig.VersionEnvVar,
+    };
+
+    private readonly string[] _values = new string[EnvironmentVariables.Length];
+
+    [OneTimeSetUp]
+    public void Setup()
+    {
+        for (int i = 0; i < EnvironmentVariables.Length; i++)
+        {
+            _values[i] = Environment.GetEnvironmentVariable(EnvironmentVariables[i]);
+            Environment.SetEnvironmentVariable(EnvironmentVariables[i], null);
+
+        }
```

Comment:
StyleCop is failing the build for this empty line.

```suggestion
            Environment.SetEnvironmentVariable(EnvironmentVariables[i], null);
        }
```